blockly-core/core/ui/block_space/block_space.js

```diff
@@ -189,6 +191,15 @@ Blockly.BlockSpace.prototype.debugLogOnEvents = function() {
   }, this);
 };
 
+/**
+ * Listens for block importing and triggers a scrollable size update.
+ * @private
+ */
+Blockly.BlockSpace.prototype.updateScrollOnBlockImport_ = function () {
+  this.events.listen(Blockly.BlockSpace.EVENTS.EVENT_BLOCKS_IMPORTED,
+      this.updateScrollableSize.bind(this));
+};
+
```

Comment:
Is it overkill to put this one line in a separate function when it's only called once?